music21/stream/base.py

```diff
@@ -10110,10 +10174,22 @@ def pitches(self) -> list[pitch.Pitch]:
         for e in self.elements:
             if isinstance(e, key.Key):
                 continue  # has .pitches but should not be added
-            # both GeneralNotes and Stream have a pitches properties; this just
-            # causes a recursive pitch gathering
-            elif isinstance(e, (note.GeneralNote, Stream)):
+            elif isinstance(e, Stream):
+                # Stream has a pitches properties; this just
+                # causes a recursive pitch gathering
                 post.extend(list(e.pitches))
+            elif isinstance(e, note.GeneralNote):
```

Comment:
disagree, please revert -- `.pitches` on Stream should not include ornamental pitches -- that would be a major change in behavior.  If people want that, they can call `score.realizeOrnaments().pitches`.

Reply:
I was wondering about that.  And in my defense I _did_ choose not to change GeneralNote.pitches to include ornamental pitches.  That seemed _way_ too much of a change.  (Hmmm... maybe otherPitches should be called ornamentalPitches.)

Reply:
Part of the reason is that most analysis ignores the ornamental notes.  So if you had three whole notes, C, C# and D, you could say that 2/3 of the notes fit in the key of C major,  but if you added `tr#` to the C#, then it'd be more like 10% do.  I wish more people would realize ornaments and expand repeats before doing analysis, but there needs to be a way that analytical methods that don't expand still work.

Reply:
Done.  I added a new stream.ornamentalPitches property that gives me the rest of what I needed in makeAccidentalsForMeasureStream (or whatever that method is called).